iphone/Classes/TiUIiOSStepper.m

```diff
@@ -28,107 +28,115 @@ -(UIStepper*)stepper
 -(void)dealloc
 {
     [stepper removeTarget:self action:@selector(stepperChanged:) forControlEvents:UIControlEventValueChanged];
-    [stepper removeTarget:self action:@selector(stepperTouch:forEvent:) forControlEvents:UIControlEventTouchDown];
+    [stepper removeTarget:self action:@selector(controlAction:forEvent:) forControlEvents:UIControlEventAllTouchEvents];
     RELEASE_TO_NIL(stepper);
     RELEASE_TO_NIL(backgroundImageCache);
     [super dealloc];
 }
 -(void)setContinuous_:(id)value
 {
+    ENSURE_TYPE(value, NSNumber);
     [[self stepper] setContinuous:[TiUtils boolValue:value]];
 }
 
 -(void)setAutorepeat_:(id)value
 {
+    ENSURE_TYPE(value, NSNumber);
     [[self stepper] setAutorepeat:[TiUtils boolValue:value]];
 }
 
 -(void)setWraps_:(id)value
 {
+    ENSURE_TYPE(value, NSNumber);
    [[self stepper] setWraps:[TiUtils boolValue:value]];
 }
 
--(void)setMaximumValue_:(id)value
+-(void)setMaximum_:(id)value
 {
-    ENSURE_TYPE(value, NSNumber)
+    ENSURE_TYPE(value, NSNumber);
     [[self stepper] setMaximumValue:[TiUtils doubleValue:value]];
 }
 
--(void)setMinimumValue_:(id)value
+-(void)setMinimum_:(id)value
 {
-    ENSURE_TYPE(value, NSNumber)
+    ENSURE_TYPE(value, NSNumber);
     [[self stepper] setMinimumValue:[TiUtils doubleValue:value]];
 }
 
 -(void)setStepValue_:(id)value
 {
-    ENSURE_TYPE(value, NSNumber)
-    NSNumber *stepValue = (NSNumber*)value;
+    ENSURE_TYPE(value, NSNumber);
+    NSNumber *stepValue = [NSNumber numberWithDouble:[TiUtils doubleValue:value]];
     if (stepValue > 0) {
         [[self stepper] setStepValue:[TiUtils doubleValue:value]];
     } else {
-        NSLog(@"[WARN]: The stepValue must be bigger than 0");
+        NSLog(@"[WARN] The stepValue must be bigger than 0.");
     }
 }
 -(void)setTintColor_:(id)value
 {
-    [[self stepper] setTintColor:[[TiUtils colorValue:value] color]];
+    TiColor *color = [TiUtils colorValue:value];
+    if (color != nil) {
+        [[self stepper] setTintColor:[[TiUtils colorValue:color] color]];
+    }
 }
 
-#pragma mark backgroundImage
--(void)setBackgroundImage_:(id)value
+-(void)setEnabled_:(id)value
 {
-    backgroundImageCache = [self imageWithImage:[self loadImage:value] fullScale:YES];
-    [[self stepper] setBackgroundImage:backgroundImageCache forState:UIControlStateNormal];
+    ENSURE_TYPE(value, NSNumber);
+    [stepper setEnabled:[TiUtils boolValue:value]];
 }
 
--(void)setBackgroundSelectedImage_:(id)value
+-(double)getValue
```

Comment:
Objective-C getter don't have the `get`-Prefix, so just `- (double)value`. Why do we need this?